cmdb.js

```diff
@@ -119,7 +119,8 @@ cmdb.prototype._fetchAll = function fetchAll(locals, url, timeout = 12000) {
 	var params = {
 		headers: {
 			apikey: self.apikey,
-			"x-api-key": self.apikey
+			"x-api-key": self.apikey,
+            forever:true
```

Comment:
This isn't a header, it's an option!

Reply:
Right apologies - changed this to how you mentioned.